docs/source/setup_osx.rst

```diff
@@ -151,26 +151,11 @@ Now exit any root environments, becoming just yourself (even though it's not
 that easy being green), for the remainder of these instructions.
 
 
-TimescaleDB 1.2.2
+TimescaleDB 1.4.1
 -----------------
 
-Homebrew::
```

Comment:
Changed/removed the Homebrew option as I couldn't (in a reasonable amount of time) identify the commands necessary to get the right version.

Reply:
yep. i had to install from source as well. Couldn't figure out how to downgrade (`brew install timescaledb@1.4.1` didn't work).